easybuild/easyconfigs/b/Boost/Boost-1.61.0-foss-2016a-Python-3.5.1.eb

```diff
@@ -0,0 +1,27 @@
+name = 'Boost'
+version = '1.61.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.boost.org/'
+description = """Boost provides free peer-reviewed portable C++ source libraries."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%%(namelower)s_%s.tar.gz' % '_'.join(version.split('.'))]
+
+patches = [
+    'Boost-%(version)s_python3.patch',
+]
```

Comment:
ok to collapse to a single line if there's only a single entry

```python
patches = ['Boost-%(version)s_python3.patch']
```

also, please include `SHA256` checksums for sources & patches, easiest way to do this is via `eb --inject-checksums` (requires EasyBuild v3.4.0), see http://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#adding-or-replacing-checksums-using-inject-checksums